src/pages/home/report/AnimatedEmptyStateBackground.tsx

```diff
@@ -37,7 +38,9 @@ function AnimatedEmptyStateBackground() {
         xOffset.value = clamp(xOffset.value + y * CONST.ANIMATION_GYROSCOPE_VALUE, -IMAGE_OFFSET_X, IMAGE_OFFSET_X);
         yOffset.value = clamp(yOffset.value - x * CONST.ANIMATION_GYROSCOPE_VALUE, -IMAGE_OFFSET_Y, IMAGE_OFFSET_Y);
         return {
-            transform: [{translateX: withSpring(-IMAGE_OFFSET_X - xOffset.value)}, {translateY: withSpring(yOffset.value)}],
+            // On Android, scroll view sub views gets clipped beyond container bounds. Set the top position so that image wouldn't get clipped
+            top: IMAGE_OFFSET_Y,
+            transform: [{translateX: withSpring(xOffset.value)}, {translateY: withSpring(yOffset.value)}, {scale: 1.15}],
```

Comment:
Here is video testing sensors on physical iOS device.

https://github.com/Expensify/App/assets/6880914/bfe040f1-0520-420d-bfcc-a270fcb87b2e

> There should be no 1px difference

I've added a top offset `top: IMAGE_OFFSET_Y,` to prevent the image from getting clipped while tilting on Android devices. This actually pushes the image down by 40px. If you think we don't need to fix it (as this is the experience on prod at the moment), I'll leave it same as before.

Reply:
Also the earlier implementation stretched out the image to 200% and then positioned the image back in view by translating the image by setting negative `IMAGE_OFFSET_X`. This caused issues on web while resizing. Instead in current implementation, I'm keeping image width at `100%` but then scale it 15% to account for the additional span required while tilting the device. 

| Staging | This Branch |
|--|--|
|![staging](https://github.com/Expensify/App/assets/6880914/aba3cea8-4aa5-40e9-9cb8-abaeeea6e2df)|![current](https://github.com/Expensify/App/assets/6880914/2bf33e5f-25e4-4e6c-97de-2d3a415264c1)|

Reply:
I think the regression was caused by [this line](https://github.com/software-mansion-labs/expensify-app-fork/blob/ee64a2983e919fa9aab7939c62cad0841b4e6ae5/src/pages/home/report/ReportActionItem.js#L630), so I would look there for a fix. I've added it, because I wanted to move the background image a bit upwards following designers' suggestion. 

I suppose that the cropped image problem can be rather related to wrong container size, not the animation itself. The animation moves the image outside of container boundaries, and as a result it gets cropped.

Reply:
@aimane-chnaif I don't see a way without modifying the animation logic. As we need to remove the '200%' width set for background image to fix the issue on web. In fact the core logic of the animation is still the same, all I have changed is using scale to position the image instead of setting 200% width and then positioning using negative margin.

We may also try generating adhoc build for this branch for extensive testing on native devices.